### ISSUE TITLE
Saltstack:salt execution module junos.install_os default installs the image on Master RE , options are not available to select dual RE , back up RE

### ISSUE THREAD

**Issue by rsmekala** [Juniper/salt#110]
### Description of Issue/Question

```
root@nms5-salt-master-b:~# salt "vmx1" junos.install_os salt://junos-install-mx-x86-64-19.3I-20190409_dev_common.0.2212.tgz no_copy=False reboot=True

vmx1:
    ----------


        Installation failed due to: "PCDATA invalid Char value 7, line 106, column 33 (line 106)"
    out:
        False
```

**Comment by rsmekala** [comment]
Problem: By default, installs on both RE. No option to specify otherwise. 

Solution: Port all PyEZ installation attributes to salt

**Comment by rsmekala** [comment]
Added all_re option. Merged to 2019.2

https://github.com/Juniper/salt/blob/04b366ca2e458b0e696e6fb6314688e7f0fa1312/salt/modules/junos.py#L1089
